### PR TITLE
Add Dan Sun to the ci member list

### PR DESCRIPTION
**Pull request by yuzisun**

### CODE REVIEW

File: ci-team.members.txt [hunk 1]
```diff
@@ -10,3 +10,4 @@ yangpa@google.com
 gongyuan@google.com
 hnalla@redhat.com
 scottleehello@gmail.com
+dsun20@bloomberg.net
```
Comment (jlewi) [inline ci-team.members.txt:13]: Please alphabetize.

Please remove

* cartick
* clarktm
* kam.d.kasravi
* kunming
* lunkai

Comment (jlewi) [reply ci-team.members.txt:13]: @scottilee Are you still planning on contributing as part of the ci-team?

Comment (yuzisun) [reply ci-team.members.txt]: removed the above list

**Comment by jlewi** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**